recipes/diffpy-structure/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "diffpy.structure" %}
+{% set version = "1.3.4" %}
+{% set sha256 = "de481329c4dc41ce1538f88fac768c8f677860602caa886fbfb3429bc17ac3ae" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/diffpy/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```

Comment:
I was pulling from github because the pypi link was acting up
`url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz` 

->

`url: https://pypi.io/packages/source/d/diffpy.structure/diffpy.structure-1.3.5.tar.gz`

see: https://circleci.com/gh/CJ-Wright/staged-recipes/70?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link